src/tests/unit_asio_stream.cpp

```diff
@@ -0,0 +1,771 @@
+/*
+* TLS ASIO Stream Wrapper
+* (C) 2018-2019 Jack Lloyd
+*     2018-2019 Hannes Rantzsch, Tim Oesterreich, Rene Meusel
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+
+#if defined(BOTAN_HAS_TLS) && defined(BOTAN_HAS_BOOST_ASIO)
+
+#include <boost/version.hpp>
+#if BOOST_VERSION >= 106600
```

Comment:
Unfortunately this is not sufficient for 'boost/beast/experimental/test/stream.hpp'. We need boost 1.68 for this. Either we stop using the test stream, or we require 1.68 for this unit test (the rest of the stream works fine with 1.66).

Reply:
I did as suggested above.